### PR TITLE
EWPP-793: Introduction field helptext change.

### PR DESCRIPTION
**Pull request by yenyasinn**

### CODE REVIEW

**Review by imanoleguskiza — DISMISSED** [review]

**Review by 22Alexandra — DISMISSED** [review]

File: tests/features/content-types.feature [hunk 1]
```diff
@@ -4,27 +4,64 @@ Feature: Common fields across the content types
...
     Examples:
-      | content_type      |
-      | Call for tenders  |
-      | Event             |
-      | Page              |
-      | News              |
-      | Policy            |
-      | Publication       |
-      | Project           |
+      | content_type       |
```
Comment (yenyasinn) [inline tests/features/content-types.feature]: Additional content types have been added.

**Review by 22Alexandra — APPROVED** [review]

**Review by 22Alexandra — DISMISSED** [review]

File: tests/features/content-types.feature [hunk 2]
```diff
@@ -4,27 +4,64 @@ Feature: Common fields across the content types
   As an editor
   I should have common fields configured the same for each content type.
 
-  Scenario Outline: Fields have the correct description.
-    Given I am logged in as a user with the "create oe_event content, create oe_news content, create oe_page content, create oe_policy content, create oe_project content, create oe_publication content, create oe_call_tenders content, access content" permission
+  Scenario Outline: Common fields have the correct description.
+    Given I am logged in as a user with the "create oe_event content, create oe_news content, create oe_page content, create oe_policy content, create oe_project content, create oe_publication content, create oe_call_tenders content, create oe_call_proposals content, create oe_organisation content, create oe_person content, create oe_consultation content, access content" permission
     When I visit "the <content_type> creation page"
-    Then I should see the text "The ideal length is 50 to 60 characters including spaces." in the "title form element"
     And I should see the text "If it must be longer, make sure you fill in a shorter version in the Alternative title field." in the "title form element"
```
Comment (imanoleguskiza) [inline tests/features/content-types.feature]: This should be moved to the "Title field have the correct description." scenario

File: tests/features/content-types.feature [hunk 3]
```diff
@@ -4,27 +4,64 @@ Feature: Common fields across the content types
...
-      | Project           |
+      | content_type       |
+      | Call for proposals |
+      | Call for tenders   |
+      | Consultation       |
+      | Event              |
+      | News               |
+      | Organisation       |
+      | Page               |
+      | Project            |
```
Comment (imanoleguskiza) [inline tests/features/content-types.feature:28]: I expected to see person here since we moved the title checks to a separate scenario

**Review by imanoleguskiza — CHANGES_REQUESTED** [review]

File: tests/features/content-types.feature (hunk 3 above)
Comment (yenyasinn) [reply tests/features/content-types.feature:28]: Yep, good point but https://github.com/openeuropa/oe_content/pull/394/files#diff-5513e603fbbd062351ac1e2b2742e9c66babeb0d8f873d4b2c474f1dd14fbf50R15 blocks us to do it. Need to merge it first.

Comment (yenyasinn) [reply tests/features/content-types.feature:28]: Person has been added.

File: tests/features/content-types.feature [hunk 4]
```diff
@@ -4,18 +4,15 @@ Feature: Common fields across the content types
...
     And I should see the text "Use this field to create an alternative title for use in the URL and in list views." in the "alternative title form element"
     And I should see the text "If the page title is longer than 60 characters, you can add a shorter title here." in the "alternative title form element"
     And I should see the text "A short overview of the information on this page. The teaser will be displayed in list views and search engine results, not on the page itself." in the "teaser form element"
     And I should see the text "Limited to 300 characters for SEO purposes." in the "teaser form element"
-    And I should see the text "A short text that will be displayed in the blue header, below the page title." in the "summary form element"
+    And I should see the text "A short text that will be displayed below the page title. This should be a brief summary of the content on the page that tells the user what information they will find on this page." in the "summary form element"
     And I should see the text "This should be a brief summary of the content on the page that tells the user what information they will find on this page." in the "summary form element"
```
Comment (drishu) [inline tests/features/content-types.feature]: This line is already tested (included) in the one above, let's remove it.

**Review by drishu — CHANGES_REQUESTED** [review]

**Review by drishu — DISMISSED** [review]

**Review by 22Alexandra — APPROVED** [review]

**Review by upchuk — APPROVED** [review]

**Pull request merged**